src/components/CreateLink.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { graphql, gql } from 'react-apollo';
+
+import { GC_USER_ID } from '../constants';
+
+class CreateLink extends Component {
+
+  state = {
+    description: '',
+    url: ''
+  };
+
+  render = () => (
+    <div>
+      <div className={'flex flex-column mt3'}>
+        <input
+          className={'mb2'}
+          value={this.state.description}
+          onChange={(e) => this.setState({ description: e.target.value })}
```

Comment:
This is a cool way to manage what the user's set so far. Another way to do it is to get the ref to the node and grab the value later on. https://reactjs.org/docs/refs-and-the-dom.html.